### DISCUSSION TITLE
Map Parts

### DISCUSSION THREAD

**Discussion by JasonGilley** [craigins/CoreKeeperMapTool#13] [Q&A] [answered]
How did you determine the map parts? Did you have some sort of reference or did you just use a map you had and determined something at a certain coordinate and then defined a map part? If that's the case, how did you match up your map to the data?

I am trying to find Ancient Feathers and add them to the list.

Thanks

**Comment by craigins** [comment, marked as answer]
The mapparts zip file is a map, character based.  it has a set of png files encoded inside, indexed off of 0,0.

The PNGs have 1 pixel per tile, the pixel color represents what the tile "is".  That is what the filter on the left uses.  For the pre-sunken sea, someone data mined the color list from the game.  The sunken sea tiles that are in the filter I manually pulled from my map after exploring a little.

Not sure what you are looking for when wanting to add Ancient Feathers to the list, I do not believe they have a pixel color associated with them.
